domain/dcsa_ovs_domain.yaml

```diff
@@ -0,0 +1,254 @@
+            UNLocationCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/UNLocationCode'
+      required:
+        - callbackUrl
+    transportCallSubscription:
+      allOf:
+        - $ref: '#/components/schemas/subscriptionID'
+        - $ref: '#/components/schemas/transportCallSubscriptionBody'
+    
+    transportCall:
+      description:  A list of all the locations involved in a transport journey.
+      type: object
+      allOf:
+        - type: object
+          properties:
+            scheduleID:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/scheduleID'
+        - type: object
+          properties:
+            carrierServiceCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/carrierServiceCode'
+        - type: object
+          properties:
+            vesselIMONumber:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/vesselIMONumber'
+        - type: object
+          properties:
+            vesselName:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/vesselName'
+        - type: object
+          properties:
+            carrierVoyageNumber:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/carrierVoyageNumber'
+        - type: object
+          properties:
+            UNLocationCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/UNLocationCode'
+        - type: object
+          properties:
+            UNLocationName:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/UNLocationName'
+        - type: object
+          properties:
+            transportCallNumber:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/transportCallNumber'
+        - type: object
+          properties:
+            facilityTypeCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/facilityTypeCode'
+        - type: object
+          properties:
+            facilityCode:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/facilityCode'
+        - type: object
+          properties:
+            otherFacility:
+              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/otherFacility'
+      required:
+        - transportCallID
+        - scheduleID
+        - carrierVoyageNumber
+        - UNLocationCode
+        - facilityTypeCode
+        - facilityCode
+    
+    ######################################
+    # Transport Event Related Objects
+    ######################################
+
+    transportEvent:
```

Comment:
Do we still want to call this "TransportEvent" when the body refers to a notification?